lib/instrumentation/modules/express.js

```diff
@@ -60,9 +61,20 @@ module.exports = function (express, agent, { version, enabled }) {
           }
         } else {
           handle = function (err, req, res, next) {
+            if (!layer.route && layerPath && typeof next === 'function') {
```

Comment:
This if-block is technically not part of the fix, but I figured that it would be best to patch the `next` function in the error handling layer just as we do in the regular layers.

Reply:
@Qard after looking more into this, I don't think this is needed in an error handler at all 🤔 I don't think it messes things up, but it just adds overhead

Reply:
Yeah...I feel like there might have been a _reason_ we didn't have that here. 🤔

Reply:
Removing it...